src/hydra/dataProvider.js

```diff
@@ -117,6 +117,7 @@ export default (
   httpClient = fetchHydra,
   apiDocumentationParser = parseHydraDocumentation,
   useEmbedded = false, // remove this parameter for 3.0 (as true)
+  disableCache = false,
```

Comment:
I think we should introduce an object for all these parameters instead of having one more parameter. WDYT?